pyartcd/pyartcd/util.py

```diff
@@ -595,9 +595,13 @@ def mail_build_failure_owners(failed_builds: dict, doozer_working: str, mail_cli
                             f'{failure["message"]}\n' \
                             f'{container_log}'
 
+        # Send email to owners of failed image builds
+        # If art is the only owner of image (example for our ci golang builder images) send instead to our default automation email
+        owner = failure['owners']
+                if (failure['owners'] and failure['owners'] != ["aos-team-art@redhat.com"])
+                else default_owner
         mail_client.send_mail(
-            to=['aos-art-automation+failed-ocp-build@redhat.com',
-                failure['owners'] if failure['owners'] else default_owner],
+            to=['aos-art-automation+failed-ocp-build@redhat.com', owner],
```

Comment:
Also `owner` can be `None`, since `default_owner` can be none, since its a non required param in ocp4. Not sure if that will cause issues with the mail_client?

Reply:
default_owner is passed via what's set in Jenkinsfile param which is our automation email. So it works fine for now, but if you think it's important to test/improve validations, we can work on that in a separate PR